lib/access_import/sprachen.rb

```diff
@@ -0,0 +1,34 @@
+class Sprachen
+  include AccUtils
+  attr_reader :records
+
+  def initialize(acdb)
+    @acdb = acdb
+    @records = hash_all
+  end
+
+  def hash_all
+    make_mappable(:tbl_Sprachen, :pk_Sprache, true)
+  end
+
+  def sanitize_record(rec)
+    rec = parse_int_fields(rec, [:pk_Sprache])
+    {
+      lang: language(rec[:t_Sprache]),
+      sprache: rec[:t_Sprache]
+    }
+  end
+
+  def language(sprache)
+    sprache = 'Tigrinja' if sprache == 'Tigrinia'
```

Comment:
I am sorry, I had found a wikipedia article in German that had it like that and I thought it was like that. Didn't know that the gem has it otherwise.